frontend/apps/reader/modules/readerlink.lua

```diff
@@ -720,6 +720,16 @@ function ReaderLink:onAddCurrentLocationToStack(show_notification)
     end
 end
 
+-- to be used on shortcuts for non-touch devices
+function ReaderLink:onAddCustomCurrentLocationToStack(show_notification)
+    self:addCurrentLocationToStack()
+    if show_notification then
+        Notification:setNotifySource(Notification.SOURCE_ALWAYS_SHOW)
```

Comment:
You can pass the source as arg to Notify, as you did previously, without set/reset.